src/Admin/Component/PerscomRankTable.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Forumify\PerscomPlugin\Admin\Component;
+
+use Forumify\Core\Component\Table\AbstractDoctrineTable;
+use Forumify\PerscomPlugin\Perscom\Entity\Rank;
+use Forumify\PerscomPlugin\Perscom\Repository\RankRepository;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveAction;
+use Symfony\UX\LiveComponent\Attribute\LiveArg;
+
+#[AsLiveComponent('PerscomRankTable', '@Forumify/components/table/table.html.twig')]
+#[IsGranted('perscom-io.admin.organization.view')]
```

Comment:
We should make dedicated perms (`organization.ranks.view`, `organization.ranks.manage`) per entity instead of a global organization.view.